lib/checks/aria/implicit-role-fallback.json

```diff
@@ -1,6 +1,7 @@
 {
 	"id": "implicit-role-fallback",
 	"evaluate": "implicit-role-fallback.js",
+	"deprecated": true,
```

Comment:
Did you mean to put this on the other rule?

Reply:
Nope, implicit-role-fallback is only used by aria-dpub-role-fallback